tools/extras/install_tensorflow_cc.sh

```diff
@@ -25,17 +25,17 @@ else
 fi
 
 
-[ ! -f bazel.zip ] && wget https://github.com/bazelbuild/bazel/releases/download/0.5.4/bazel-0.5.4-dist.zip -O bazel.zip
+[ ! -f bazel.zip ] && wget https://github.com/bazelbuild/bazel/releases/download/0.15.0/bazel-0.15.0-dist.zip -O bazel.zip
```

Comment:
I have rather a general question about this file. bazel is widely available on many distros package feeds. My (Ubuntu 18.04) offers me v0.24.1, this installs v0.15.1. And it depends on jdk, whoch I do not have (it's also packaged, but I would need it only to build bazel to build tensorflow, if I am not misunderstanding? Maybe we should check if (a) bazel is installed on the system and, if not, (b) mention that it might be available to install? Certainly in the section that complain about a missing jdk. Here we kind of unconditionally possibly too much unnecessary stuff that is likely available otherwise. I may well be mistaken, just from a superficial look at it.

Reply:
I initially tried to use the newest bazel version but tensorflow complained that the version was too new. It would be possible to check if the correct bazel version is on the path but I chose to follow the tensorflow documentation regarding tested installs and make minimal changes to the script.

Reply:
Oh, I see. Complained that bazel was too new, wow! Thanks for the clarification.

Reply:
If I remember correctly it requested to downgrade to at most 0.21 but I can double check again.

Reply:
Up to you entirely, thanks for this contribution in any case! Personally I usually attempt to make the setup scripts as painless to the user as reasonable, but then I won't object to your definition of "reasonable" in this sense :)

Oh, and I just checked what versions are available to me, and I think I got actually seriously confused. First of all, I have the Google's deb feed for bazel added, probably at some time in the past, and forgot about it. So it is not at the Ubuntu's feed. And when I checked their Package.gz manifests, by downloading the file directly, it seems that the only version available at the feed for 18.04 is Bazel 0.24.1. Looks like they either unpublish the old ones, or did not possibly publish anything older than that for 18.04 ever. So the attempt to match the installed version of Bazel looks now more like a shot in the dark; the user (well, at the least, this particular user) has not even an option to downgrade his Bazel to the required version.

So I suggest just leaving this thing as is, do not spend much time on that please. I'm sorry about this confusion! I actually just meant to ask a question; I did not realize that tf was so picky about the version of blaze it uses.

Reply:
I initially worked on this because some of my colleagues wanted to try out the lattice rescoring with a custom tensorflow language model but we ended up using the recipe provided in the Kaldi recipes. Maybe it is best to recommend that approach to most of the users anyway.